client/src/app/videos/+video-watch/video-watch.component.ts

```diff
@@ -140,6 +141,11 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
       }, undefined, this.i18n('Subscribe to the account'))
     ]
     if (this.isUserLoggedIn()) this.hotkeysService.add(this.hotkeys)
+
+    if (this.user.pipPlayer) {
```

Comment:
`this.user` is not defined at that point.

Reply:
I can test my code again, but before pushing the last commit, my code worked fine and the setting was respected :confused:

Reply:
You mean commit https://github.com/Chocobozzz/PeerTube/pull/1345/commits/e58cd73f914c8da31d5597942510d9d40113083d is at fault?

Reply:
What you said was ambiguous. I thought "before pushing the last commit" = "the commit before the last one".

Ping me whenever :slightly_smiling_face:

Reply:
the code works fine, tested it again.
The setting is respected and no error is logged to the console.

Reply:
Ok, I'll try again later tonight then.

What is your browser/node version?